docs/SaaS/dpsaas/api.md

```diff
@@ -246,4 +247,25 @@ curl -k https://bc-saas.172.22.96.209.nip.io/basic/depositories/18b682430b2ee5cd
 
 ```json
 {"index":"6","kid":"18b682430b2ee5cdcb17c0ea678a52223056d256","platform":"bestchains","operator":"0x02624bfc011381aa1f4e19db60e3f46673f1f171","owner":"0xe4af6dcf67b5ec06d903ec175a0c7e47f38249b2","blockNumber":72,"name":"abc","contentName":"","contentID":"id","contentType":"id","trustedTimestamp":"1683343692"}
-```
\ No newline at end of file
+```
+
+## 根据存证 kid 获取存证证书
+
+`API`: GET /basic/depositories/certificate/:kid
+
+`测试`:
+
+```
+curl -k https://bc-saas.172.22.96.209.nip.io/basic/depositories/certificate/18b682430b2ee5cdcb17c0ea678a52223056d256
+```
+
+`返回`:
+
+证书文件 `{kid}.pdf` 会从浏览器中直接下载
```

Comment:
Do we need a picture to show what this `kind.pdf` look like?

Reply:
👌